misk/build.gradle

```diff
@@ -10,7 +9,7 @@ kotlin {
 apply plugin: 'org.junit.platform.gradle.plugin'
 apply plugin: 'com.vanniktech.maven.publish'
 apply from: rootProject.file("gradle/localdb.gradle")
-apply from: rootProject.file("gradle/web.gradle")
+apply from: "https://raw.githubusercontent.com/square/misk-web/master/gradle/web.gradle"
```

Comment:
How come we still need the `web` task?

Reply:
Misk ships with config, example, and loader tabs and still needs web task to compile that code.

Reply:
And both projects need the exact same task without changing any configs?